web/deeamon128/html/signup-form/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/47ce19f1c7.js" crossorigin="anonymous"></script>
+  </script>
```

Comment:
you missed the start tag for "</script>"

Reply:
my bad. it was from a previous code that I have deleted. thanks a lot.